Update readme instructions 2 (#86)
XRPLBounties/Proposals: README.md

Approved bounties can be found above as `.md` (Markdown) files in the `bounties` folder in this repo.

(Open Pull Requests are NOT approved bounties - they are still pending review and approval.)

> **Please hold off on starting any work for bounty proposals until they have been officially merged, as some bounty PRs might not receive financial approval. We want to ensure that all of your hard work is properly compensated**

Reviewer: ```suggestion
**Please hold off on starting any work for bounty proposals until they have been officially merged, as some bounty PRs might not receive financial approval. There are no bounty funds available for any proposal PR that has not been merged.**
```
